pos_product_category/point_of_sale_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="product_template_form_view" model="ir.ui.view">
+            <field name="name">product.template.form.inherit</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_form_view"/>
+            <field name="priority" eval="100"/>
+            <field name="arch" type="xml">
+                <field name="pos_categ_id" position="replace"/>
```

Comment:
Can you make the field invisible instead of replacing it?
It will avoid problem if an other module base his field position on this field